client/src/components/BookEvent/index.js

```diff
@@ -27,26 +26,32 @@ class BookEvent extends React.Component {
   handleSubmitForm = event => {
     event.preventDefault();
     const { history } = this.props;
+    const { serviceId } = this.props.match.params;
     axios
-      .post("/event", this.state)
+      .post(`/event/${serviceId}`, this.state)
       .then(data => {
-        history.push("/events");
+        history.push(`/events/${serviceId}`);
       })
-      .catch(error => console.log(error));
+      .catch(error => {
+        const { history } = this.props;
```

Comment:
why did you identify ```history``` twice! you can use the first declaration for both

Reply:
removed